src/components/Whiteboard/ElementBehaviors/Selection/ElementBorder.tsx

```diff
@@ -116,7 +116,7 @@ export function ElementBorder({ elementIds, padding = 1 }: ElementBorderProps) {
                 borderWidth={selectionBorderWidth}
               />
             </>
-          ) : (
+          ) : elements[0].type !== 'image' ? (
```

Comment:
I am wondering: does it makes sense to do resizing already in this PR? To see how changed code fits together.